### PR TITLE
Feature add static content

### PR DESCRIPTION
**Pull request by tornad**
Here's the possibility to add static content to the fake api server. I use fake api for small module testing, and I need to serv html demo content at the same time than the faked api.  
Review welcome if you want some changes...

### CODE REVIEW

**Comment by couchand** [comment]
This seems like a valuable feature.

I'm not sure that the `console.log` is all that valuable.  I'm guessing you added it to mirror the log below in the `listen` method, which now that I look at it, probably belongs in the example files and not in the library at all.  Your thoughts on that?

File: README.markdown [hunk 1]
```diff
@@ -170,6 +170,10 @@ new **Server**()
   * register a given resource for the API.  the appropriate REST
     verbs will be routed.
 
+**static**(*path*)
+
+  * add a static path to the server for optional static contents(demo html, assets)
```
Comment (couchand) [inline README.markdown]: Please limit the line length to 70 characters here.  It's arbitrary but it's how the rest of the README is formatted.

**Comment by couchand** [comment]
:notebook: Note: this will require bumping the minor package version number.

**Comment by couchand** [comment]
One other thought: could you add a quick test for this to `tests/integration`?

**Comment by tornad** [comment]
I modified the PR with your recommandations.

**Comment by tornad** [comment]
Any News ? :)

**Comment by couchand** [comment]
My apologies, I was out on vacation.

Thanks for the work!  :clap:

**Pull request merged**